Header header
int32[2] object_id
<<<<<<< HEAD
float32[2] grade_certainty
=======
float32[2] ratio
>>>>>>> 982197c1772db2b0512905277958f7444f086853

